qpylib/app_qpylib.py

```diff
@@ -9,7 +9,10 @@
 Q_CACHED_MANIFEST = None
 
 def get_app_id():
-    return get_manifest_field_value('app_id', 0)
+    app_id = os.getenv('QRADAR_APP_ID')
+    if app_id is None:
+        app_id = 0
+    return str(app_id)
```

Comment:
You've changed behaviour here. get_app_id should return an int, not a string.

Reply:
Also, just noticed a bug in get_app_base_url.   
`if app_id == ''` should be `if app_id == 0`